### ISSUE TITLE
Drop Inline Tab Behavior

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#6061]
### Expected Behavior
If a drop is inline when you tab beyond the drop the focus should move to the next element

### Actual Behavior
The focus is not moving to the next element.

### URL, screen shot, or Codepen exhibiting the issue
See https://github.com/grommet/grommet/pull/6049

### Steps to Reproduce

Tab to an inline Drop and then try to continue tabbing

### Your Environment

- Grommet version: stable branch
- Browser Name and version: chrome
- Operating System and version (desktop or mobile):

**Comment by ShimiSun** [comment]
https://github.com/grommet/grommet/issues/5896  is involving two accessibility issues of Select and for the Drop. I think we can keep the issue to track the Select improvement, while the Drop accessibility issue could be tracked here as part of the Drop `inline` solution.

**Comment by NeversSync** [comment]
Seconding this issue, I am seeing tab-index="-1" added to all Drop child elements when I add the inline prop, which breaks tab functionality. 
Any updates here?

**Comment by jcfilben** [comment]
Hey @NeversSync no recent updates on this, but you can take a peek at https://github.com/grommet/grommet/pull/6078 to see some more discussion related to this

**Comment by jcfilben** [comment]
The trapFocus behavior was improved in https://github.com/grommet/grommet/pull/7324.
For inline drops `trapFocus={false}` should be used so that the focus does not get trapped within the drop.
